telemetry/specs/INT.mdk

```diff
@@ -384,12 +384,32 @@ ignore the designated bit position.)
 * TCP/UDP destination Port field: A port number in layer 4 will be reserved to
 indicate the existence of INT after TCP/UDP. INT source changes the destination
 Port, and sink must restore the original port number.
+* New Probe Marker fields: arbitrary 32-bit values are inserted after IP TCP/UDP header
```

Comment:
I think you mean 64-bit values?
"after TCP/UDP header, before INT Shim header"

Reply:
And how about citing the IETF draft of DPP?

Reply:
I just added the IETF draft links, also corrected the bits